adoc/helm_chart_override.adoc

```diff
@@ -0,0 +1,417 @@
+include::attributes.adoc[]
+
+[#pr-helm-chart-override]
+== Overriding the {productname} {helm} chart
+
+The {productname} ({productnameshort}) {helm} chart is delivered with default values.
+You can adjust the {helm} chart installation by one of the following ways:
+
+* Append specific parameters to the `--set` flags on the `helm install` command line, for example:
+[source]
+  $ helm install `RELEASE_NAME` oci://registry.suse.com/private-registry/private-registry-helm \
+  --set harborAdminPassword=<MY_PASSWORD> \
+  --set externalURL=https://my-harbor.domain
+* Create a custom `my-values.yaml` file and pass it to the `--values` flag, for example:
+[source]
+  $ helm install `RELEASE_NAME` oci://registry.suse.com/private-registry/private-registry-helm \
+  --values my-values.yaml
+
+[#pr-helm-chart-override-parameters]
+=== Overriding {helm} chart parameters and values
+
+The following tables list all parameters with descriptions that you can use to override the default installation values.
+
+.Global parameters
+[options="header"]
+|===
+|Name |Description |Value
+|`global.imageRegistry` |Sets a global override for the container image registry used for all images. |`""`
+|`global.imagePullSecrets` |Sets global pull secrets for accessing the container image registry. |`[]`
+|===
+
```

Comment:
Would it be possible to delete the values of all parameters? It is enough that the parameters are displayed.

```suggestion
|Name |Description 
|`global.imageRegistry` |Sets a global override for the container image registry used for all images.
|`global.imagePullSecrets` |Sets global pull secrets for accessing the container image registry. 
|===

```